src/features/concert/hooks/useExpectations.js

```
@@ -0,0 +1,405 @@
+    }
+  }, [concertId, currentPage, pageSize]); // 이 값들이 변경되면 함수 재생성
+
+  /**
+   * 새로운 기대평을 작성하는 함수
+   *
+   * @param {import('../types/expectation.js').ExpectationFormData} expectationData - 작성할 기대평 데이터
+   * @returns {Promise<import('../types/expectation.js').ExpectationReview>} 생성된 기대평 정보
+   */
+  const createExpectation = useCallback(async (expectationData) => {
+    try {
+      // concertId 유효성 검증
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)